packages/wasmfs/src/index.ts

```diff
@@ -23,23 +33,97 @@ export default class WasmFsDefault {
     });
   }
 
-  toJSON() {
-    return this.volume.toJSON();
+  private _toJSON(
```

Comment:
This are based on the original implementation of memfs.

https://github.com/streamich/memfs/blob/master/src/volume.ts#L805-L870

It doesn't let us extend it (because they are private), so we need to reimplement.

Reply:
I also added the `encoding` argument to the function :)

Reply:
Sounds good 😄